Library/ViewModels/DashboardReferrerRowStackViewViewModel.swift

```diff
@@ -38,15 +38,15 @@ public final class DashboardReferrerRowStackViewViewModel: DashboardReferrerRowS
 
     self.pledgedText = countryReferrer
       .map { country, referrer in
-        Format.currency(referrer.pledged, country: country) + " ("
+        Format.currency(Int(referrer.pledged), country: country) + " ("
```

Comment:
This is fine. It's a way of `floor`ing before rendering. The main issue was here, where we were summing the numbers up, which could result in a much lower number than expected: https://github.com/kickstarter/ios-oss/pull/225/files#diff-e9f9d9570eaf3e04d6369eafbe3c991cR116